_episodes/12-cluster.md

```diff
@@ -1,455 +1,38 @@
 ---
-title: "Working on a remote HPC system"
+title: "Exploring Remote Resources"
 teaching: 25
 exercises: 10
 questions:
-- "What is an HPC system?"
-- "How does an HPC system work?"
-- "How do I log in to a remote HPC system?"
+- "How does my local computer compare to the remote systems?"
+- "How does the login node compare to the compute nodes?"
+- "Are all compute nodes alike?"
 objectives:
-- "Connect to a remote HPC system."
-- "Understand the general HPC system architecture."
+- "Survey system resources using `nproc`, `free`, and the queuing system"
+- "Compare & contrast resources on the local machine, login node, and worker
+nodes"
+- "Learn about the various filesystems on the cluster using `df`"
+- "See `who` else is logged in"
+- "Glance at the number of idle and occupied nodes"
```

Comment:
```suggestion
- "Assess the number of idle and occupied nodes"
```

As far as I can tell, an exploration of the output of these kinds of monitoring commands is not really written into the episode? Assuming that the Instructor will take time to explain this in a workshop setting, to help learners understand how the output should be parsed, I think "assess" represents the skill they are learning fairly well.

Reply:
In the interests of security, some HPCs are starting to block a user's view of system use, e.g. `qstat`/`squeue` only show the user's own jobs. It may be necessary to tune the lesson to show whatever information/visualisation for the local system may be available. I think the intent here is perhaps along the lines of "Learn about shared system utilisation"?

Reply:
Right, this episode looks at the output of `sinfo` or `pbsnodes` to see if any nodes are idle. I agree that "assess" is a good descriptor. This is not a job- or user-level inspection, so hopefully skirts the security controls?